.github/workflows/main.yml

```diff
@@ -0,0 +1,28 @@
+name: RPC Tests
+
+on:
+  pull_request:
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix: 
+       os: [ubuntu-latest, macos-latest, windows-latest]
+             
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js 14.x
+        uses: actions/setup-node@v1
+        with:
```

Comment:
It seems like we don't have a node version in `package.json`? Possibly omit to use whatever version of node is on the OS.